docs/src/start/software-management.md

```diff
@@ -41,7 +41,7 @@ Managing the software packages installed on the devices from your Cumulocity ten
 * Populate the software repository with all the software packages and versions you plan to install.
 * Trigger software update operations on the devices, to install specific packages from the repository.
 
-Find more information about [managing the device software](https://cumulocity.com/guides/users-guide/device-management/#managing-device-software)
+Find more information about [managing the device software](https://cumulocity.com/docs/device-management-application/managing-device-data/#managing-software)
```

Comment:
Noticing that this not a simple `s|guides/users-guide|docs|g` update, do we have a tool to check broken links?

Reply:
Unfortunately no. The algolia crawler will detect this after the fact (as a worst case scenario), however I think someone needs to verify that the contents of the new links also makes sense (as some were not so obvious from the link names).

Reply:
Though it is good you commented here as it made me have a look around for tooling to validate external links. I did a test run of https://github.com/becheran/mlc, and whilst not perfect it did find a broken c8y doc link!

I'll evaluate adding a broken link checker into the CI pipeline (but there are still a few things to consider before doing so).